spec/cc/engine/lockfile_specs_spec.rb

```diff
@@ -20,16 +20,8 @@ module CC::Engine
         @specs.include?("python").must_equal(false)
       end
 
-      it "returns false when a LockfileError occurs while parsing" do
-        fixture = "./spec/fixtures/Gemfile.lock.with_merge_conflicts"
-        specs = LockfileSpecs.new(fixture)
-
-        specs.include?("ansi").must_equal(false)
-      end
-
       it "returns false when no Gemfile.lock file is found" do
         specs = LockfileSpecs.new(SecureRandom.hex(32))
-
```

Comment:
:lipstick: I like this line. Our styleguide says to separate the 4 phases of specs with whitespace.